contracts/multisig/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+
+# Changelog
+
+## [Unreleased](https://github.com/axelarnetwork/axelar-amplifier/tree/HEAD)
+
+[Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/multisig-v2.0.0..HEAD)
+
+## [v2.0.0](https://github.com/axelarnetwork/axelar-amplifier/tree/multisig-v2.0.0) (2025-05-09)
+
+[Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/multisig-v1.2.1..ampd-v2.0.0)
```

Comment:
The base being used in the URL is ampd instead of multisig version. You can compare changes within a folder between two refs I think

Reply:
switched, i could not find such a way to compare only changes in a folder